contract/data/contract_cron.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding='UTF-8'?>
-<odoo>
+<?xml version="1.0"?>
```

Comment:
You must keep the encoding for XML files. It's Python 3 where that's no more needed, but XML is not Python